homeassistant/components/light/xiaomi_miio.py

```diff
@@ -96,11 +114,15 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
 
     _LOGGER.info("Initializing with host %s (token %s...)", host, token[:5])
 
+    devices = []
+    unique_id = None
+
     if model is None:
         try:
             miio_device = Device(host, token)
             device_info = miio_device.info()
             model = device_info.model
+            unique_id = "{}-{}".format(model, device_info.mac_address)
```

Comment:
why include model and not just mac address ? Mac address should be unique per device.

Reply:
Oh I see the primary/secondary device things now.